spec/lib/bank_accounts/account_spec.rb

```diff
@@ -0,0 +1,58 @@
+require './spec/spec_helper'
+require './lib/bank_accounts/account'
+
+describe BankAccounts::Account do
+
+
+  describe "class methods" do
```

Comment:
When using the describe blocks, you really want to use them to describe the behavior, rather than the code.